### PR TITLE
Fix blitz install for recipes that use the path helper

### PR DESCRIPTION
**Pull request by dillonraphael**
Closes: #3900

### What are the changes and their implications?

Check if `src/pages` directory exists in the installer paths helper, else use the root `pages` directory.

## Bug Checklist

- [X] Changeset added (run `pnpm changeset` in the root directory)

### CODE REVIEW

File: packages/blitz/src/installer/utils/paths.ts [hunk 1]
```diff
@@ -18,12 +18,23 @@ function getBlitzPath(type: string) {
...
 }
 
+function findPageDir() {
+  const srcPagePath = `src/pages`
+  const srcPage = fs.existsSync(path.resolve(srcPagePath))
+
+  if (srcPage) {
+    return srcPagePath
+  } else {
+    return `pages`
```
Comment (maotora) [inline packages/blitz/src/installer/utils/paths.ts:28]: I think previous versions of blitz were using `app/pages` instead of just `pages`.

**Comment by maotora** [comment]
@dillonraphael Hey nice catch, I was facing the same error trying to install the tailwind recipe. 
I just left a comment in one of your files. I think previous versions of blitz were using `app/pages` or something similar instead of just `pages`.

**Comment by maotora** [comment]
Thanks for your response, @dillonraphael!  

I just tested this and I found there is another edge case to cover for Tailwindcss to get our coding files which no longer reside at pages or app directories. 
We need to update `tailwind.config.js` content array to reflect the directory change as well.

I came up with something like this which fixed it;

```js
// tailwind.config.js
module.exports = {
  content: ["src/**/*.{js,ts,jsx,tsx}"],
  theme: {
    extend: {},
  },
  plugins: [],
}
```

Which worked!

**Comment by dillonraphael** [comment]
> 

Good find @maotora. Would you like to open another PR for updating the tailwind recipe (since this is tailwind recipe specific)?

**Comment by maotora** [comment]
@dillonraphael sure I can do that. Will make a PR and tag you then.

**Comment by maotora** [comment]
@dillonraphael done I've made a PR: [here](https://github.com/blitz-js/blitz/pull/3997)

**Comment by cassilup** [comment]
/Still experiencing this in 2.0.2 (released last week).

**Comment by Doc0x1** [comment]
Still experiencing this in 2.0.3 as well. I haven't tested to make sure but I think this might be an issue for projects that are using the App Router instead of the older Pages system.

**Comment by siddhsuresh** [comment]
yeah @Doc0x1, that seems likely, we would need to add additional handling to also check for the existence of the `app` directory in the same area as the change in this PR.

**Pull request merged**